crate_universe/src/rendering.rs

```diff
@@ -33,6 +33,7 @@ pub struct Renderer {
     config: RenderConfig,
     supported_platform_triples: BTreeSet<TargetTriple>,
     generate_target_compatible_with: bool,
+    alias_rule: AliasRule,
```

Comment:
This feels like it should live in `RenderConfig` and be called `default_alias_rule`?